src/Api.js

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import {AlbumScript} from "./Script";
+import {HomeScript} from "./Script";
+import {SearchScript} from "./Script";
+
+export class Api {
```

Comment:
Класс и его методы не задокументированы.
```suggestion
/**
* Класс, взаимодействующий с api спотифая
*/
export class Api {
```